tests/unit/loadDojoConfig.ts

```diff
@@ -0,0 +1,18 @@
+import { describe, it } from 'intern!bdd';
+import * as assert from 'intern/chai!assert';
+import loadDojoConfig from '../../src/loadDojoConfig';
+
+describe('loadDojoConfig', () => {
+	it('should load the specified module if it exists', () => {
+		const config = loadDojoConfig('tests/support/mocks/dojo.config.js');
```

Comment:
I've raised an issue here https://github.com/dojo/cli/issues/88, to solve this generically in cli - let me know if that makes sense.